src/main/java/omero/cmd/admin/UpdateSessionTimeoutRequestI.java

```diff
@@ -143,6 +153,16 @@ protected Session updateSession() {
         }
 
         boolean isAdmin = current.getCurrentEventContext().isCurrentUserAdmin();
+        if (!isAdmin && maxUserTimeToLive != 0 && timeToLive != null
+                && timeToLive.getValue() > maxUserTimeToLive) {
+            timeToLive = omero.rtypes.rlong(maxUserTimeToLive);
+            helper.info("Attempt to modify timeToLive beyond maximum");
+        }
+        if (!isAdmin && timeToIdle != null
```

Comment:
not check for ``maxUserTimeToIdle > 0``

Reply:
That behaviour is undefined, the implementation here is in keeping with the previous one. I don't think we're declaring anywhere scenarios where we're allowing sessions to never timeout. There are not test cases or design coverage for such a implementation to my knowledge.

Reply:
understood